pkgs/development/libraries/teensy-cmake-macros/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config }:
+
+stdenv.mkDerivation {
+  pname = "teensy-cmake-macros";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "newdigate";
+    repo = "teensy-cmake-macros";
+    rev = "dc401ed23e6e13a9db3cd2a65f611a4738df3b0e";
```

Comment:
Why aren't you using `version` here?

Reply:
The upstream repo isn't tagged unfortunately. Is there a better way to represent that in the package version?

Reply:
Unfortunately, no one that I know.